### PR TITLE
official profiles for Geeetech printers

### PR DESCRIPTION
**Pull request by whoseyoung**
Re-modify and add some series of printer parameter configurations.

24 Geeetech printers, including 9 series models.
Thunder series. A high-speed printer and the maximum printing speed can reach 300 mm/s.
Mizar series. A machine that has very exquisite print quality and ultra quiet printing.
Other Geeetech printers.
---Request from Geeetech offical . All relevant tests have passed.

### CODE REVIEW

**Comment by whoseyoung** [comment]
All issue items processed.

**Comment by whoseyoung** [comment]
Changes requested by the bot have been made, and conversations resolved. Let me know if any additional items are needed or if there are any questions.

**Comment by jellespijker** [comment]
Can't merge this yet, since it has failing UnitTests, See https://github.com/Ultimaker/Cura/actions/runs/3664838630/jobs/6195513166#step:13:549

**Review by jellespijker — CHANGES_REQUESTED** [review]
Check the failing UnitTests

File: resources/definitions/geeetech_Thunder.def.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        "cool_min_layer_time":{ "value": "1.3 if speed_infill>=200  else 2.5" },
+	    "cool_fan_full_at_height": { "label": "Regular Fan Speed at Height","value": "layer_height_0 +  layer_height" },
+		"cool_fan_speed":{ "maximum_value": "151","value": "151 if speed_infill>=200  else 100" },
+		"cool_fan_speed_min":{ "maximum_value": "151","value": "cool_fan_speed" },
+		"cool_fan_speed_max":{ "maximum_value": "151","value": "cool_fan_speed" },  		
+
+        "material_print_temperature": { "value":  "200 if speed_infill <=150 else 205 if speed_infill <= 200 else 215 if speed_infill <= 260 else 220","maximum_value": "250" },
+        "material_print_temperature_layer_0": { "value": "material_print_temperature" },
+		"material_initial_print_temperature": { "value": "material_print_temperature - 5" },
+		"material_flow_layer_0": { "value": 95 },
```
[the remark this answers is not in the document]
Comment (whoseyoung) [reply resources/definitions/geeetech_Thunder.def.json]: fixed the redundant-override problem.

**Comment by jellespijker** [comment]
LGTM;

Thank you for your contribution, will merge this to main, they should be available in Cura 5.3

**Review by jellespijker — APPROVED** [review]

**Comment by whoseyoung** [comment]
After Cura5.3.1, it was found that this configuration was merged or replaced. Please help restore this configuration file, thank you

**Pull request merged**